src/mocks/entity/index.ts

```diff
@@ -0,0 +1,12 @@
+import mockDatasetData from './mockDataset'
+import { MockEntityData } from './MockEntityData'
+import mockFileEntityData from './mockFileEntity'
+import mockProjectEntityData from './mockProject'
+
+const mockEntities: MockEntityData[] = [
+  mockFileEntityData,
+  mockProjectEntityData,
+  mockDatasetData,
+]
```

Comment:
Similar to what we already do for mock user profiles, construct a set of mock entities that MSW can pluck from by ID